### PR TITLE
Update frequently-asked-questions.rst

### LINKED ISSUE #4339: Typo in FAQ documentation
In the section [https://solidity.readthedocs.io/en/v0.4.24/frequently-asked-questions.html#what-happens-to-a-struct-s-mapping-when-copying-over-a-struct](url), there is a typo.

> In this case, the mapping of the struct being copied over into the userList is ignored as there is no “list of mapped keys”. Therefore it is not possible to find out which values should be copied over.

Should read:
> In this case, the mapping of the struct being copied over into **user2** is ignored as there is no “list of mapped keys”. Therefore it is not possible to find out which values should be copied over.

### PR DESCRIPTION
**Pull request by aj07**
Fixes #4339

### CODE REVIEW

**Comment by axic** [comment]
Can you please use more meaningful commit messages? :)

File: docs/frequently-asked-questions.rst [hunk 1]
```diff
@@ -414,7 +414,7 @@ This is a very interesting question. Suppose that we have a contract field set u
        User user2 = user1;
     }
 
-In this case, the mapping of the struct being copied over into the userList is ignored as there is no "list of mapped keys".
+In this case, the mapping of the struct being copied over into the user2 is ignored as there is no "list of mapped keys".
```
Comment (axic) [inline docs/frequently-asked-questions.rst]: `user2` should be enclosed in backticks.

**Comment by chriseth** [comment]
Please remove `the` and squash the commits.

**Comment by chriseth** [comment]
I'm really sorry, but reviewing this over and over again this is taking too many resources.

**Pull request closed without merging**